src/testing/complete-redux-store-dump.js

```diff
@@ -21,7 +21,8 @@ export default {
           'display-name': 'eggman',
           identity: 'user-eggman@external',
           'controller-access': '',
-          'model-access': ''
+          'model-access': '',
```

Comment:
There were some erroneous data in this dump that became apparent once I set the correct type for `ModelData`.